morphocut/str.py

```diff
@@ -12,7 +12,7 @@ class Format(Node):
     Format a string just like :py:meth:`str.format`.
 
     This class allows you to create and customize your own string formatting behaviors using the same 
-    implementation as the :py:meth:`str.format` methods.
+    implementation as the :ref:`Python Format String Syntax <python:formatstrings>` methods.
```

Comment:
Replace whole summary and description with:
``Format strings like :py:meth:`str.format` using the the :ref:`Python Format String Syntax <python:formatstrings>`.``